script.js

```diff
@@ -72,21 +67,50 @@ document.getElementById("dropdrop").addEventListener("mouseout", offmouse);
 
 //randomCity()
 
-
-   
-function getRandomCity(list,items =1 ) {
-    console.log(city)
-    [...list].sort(() => Math.random() > 0.5 ? 1: -1).slice(0,items)

```

Comment:
[...list] was causing an error (syntax error I think). The call to getRandomCity on line 85 also wasn't passing a city argument.